extensions/grpc/runtime/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-grpc-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <vertx.version>3.9.0</vertx.version>
+    </properties>
+
+    <artifactId>quarkus-grpc</artifactId>
+    <name>Quarkus - gRPC - Runtime</name>
+    <description>gRPC support for Quarkus</description>
```

Comment:
This should a sentence starting with a verb as it's what we will have on code.quarkus.io ("Serialize..." "Connect to...")

Reply:
Let's go with "Serve and consume gRPC services"